ansible/roles/csm.ncn.enable_chn/defaults/main.yml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Hewlett Packard Enterprise Development LP
```

Comment:
Need 2022 - format is incorrect, but looks like the license checker is skipping this file...

Reply:
Fixed